trappy/thermal.py

```diff
@@ -37,8 +37,8 @@ class Thermal(Base):
     """The Pivot along which the data is orthogonal"""
 
     def plot_temperature(self, control_temperature=None, title="", width=None,
-                         height=None, ylim="range", ax=None, legend_label="",
-                         tz_id=None):
+                         height=None, xlim="default", ylim="range", ax=None,
+                         legend_label="", tz_id=None):
```

Comment:
Need to update the docstring

Reply:
For some reason the `ylim` wasn't referenced in the docstring, so I didn't reference the `xlim` parameter either. I suppose it'd be better if I referenced both ?

Reply:
Yeah might as well fix it while you'r'e touching this code